src/Unleash/Unleash.csproj

```diff
@@ -1,13 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <!--
   <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' ">
     <TargetFrameworks>netstandard2.0;net48;net472;net47;net461;net46;net451;net45</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(OS)' != 'Windows_NT' ">
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
+  </PropertyGroup>-->
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This needs changing in Yggdrasil.Engine first before we can go back to shipping more than just .NET 6 binaries